### ISSUE TITLE
Being able to follow the crawling process

### ISSUE THREAD

**Issue by qdequele** [meilisearch/scrapix#5]
Implement a WebSocket that would allow following the process of a crawling request.

**Comment by qdequele** [comment]
This may be solved by the webhooks?!

**Comment by brunoocasali** [comment]
Yes, that's the idea! In fact, we don't need to "watch" the process, we just need to be informed when something happens in the process (an error or a finished event). So using webhooks will do the job!

**Comment by qdequele** [comment]
I've created a PR for this #62, but the test is not working; I don't know why. I didn't have the time to check the test yet.
